examples/reaching3d_sac_kukalwr4.gin

```diff
@@ -0,0 +1,23 @@
+# Training & playing the reaching3d task with Sac algorithm
+#
+# DISPLAY=null python -m alf.bin.train --root_dir=~/tmp/reaching3d --gin_file=reaching3d_sac_kukalwr4.gin
+# python -m alf.bin.play --root_dir=~/tmp/reaching3d --gin_file=reaching3d_sac_kukalwr4.gin
+
+include 'common_nonimage_sac.gin'
+import social_bot
+
+# environment config
+create_environment.env_name="SocialBot-PlayGround-v0"
+create_environment.num_parallel_environments=12
+create_environment.env_load_fn=@suite_socialbot.load
+
+PlayGround.agent_type="kuka_lwr_4plus"
+PlayGround.tasks=[@Reaching3D]
+Reaching3D.reward_shaping=True
+PlayGround.step_time=0.05  # 20Hz step frequence
+PlayGround.max_steps=50  # 2.5s
+# make sure max_episode_steps of gym does not override PlayGround.max_steps
```

Comment:
Do you mean max_episode_steps of gym should be larger than PlayGround.max_steps?

Reply:
Yes. So that the expected return of last step of an episode is correctly handled.

Reply:
Possible situations:

If max_episode_steps is reached and the env is reset, the potential discounted future reward will be take into account when update the value network. If not ended in purpose, most task should be like this by default . (max_episode_steps < max_steps)

If max_steps is reached with 'done' and the env is reset, the algorithm don't know env reaches time limits and treat this the same as done. All potential discounted reward in the future will be considered as zero. For some task like goal task and reaching task we use this way. (max_episode_steps > max_steps with done for each ending case)

If max_steps is reached without 'done' and the env is reset, the algorithm don't know env is reset, the initial state of next episode will be considered as the next state of the last step. This should never happen. (max_episode_steps > max_steps without done)